src/server/subRound.js

```diff
@@ -24,17 +24,21 @@ export class SubRound {
   }
 
   getCaptions() {
-    return this.captions;
+    return this.captions.map((caption) => {
+      return {
+        playerName: caption.player.name,
```

Comment:
We should use the `getName` method instead of getting name directly

Reply:
done